packages/mockserver-config-writer/src/mockserver-config/package-json.ts

```diff
@@ -103,6 +127,25 @@ function replaceConfig(startScript: string, configStartIndex: number): string {
     return startMockScript;
 }
 
+/**
+ * Remove mockserver middleware from ui5.dependencies in package.json. If no ui5.dependencies
+ * are left, remove the complete property dependencies or respective also the ui5 property.
+ *
+ * @param packageJson - parsed package.json content
+ */
+function removeMockserverUi5Dependencies(packageJson: Package) {
```

Comment:
Could we please change it to `removeUi5Dependencies` and remove the whole block because it is not just the mockserver that isn't required there anymore, correct?

Reply:
I would rather keep the change local to the mockserver scope the command is meant for, and not do a general cleanup for CLI 3, as there might be other entries the user wants to review, even though the dependency is not used anymore (but also poses no harm iirc).

Reply:
> Could we please change it to `removeUi5Dependencies` and remove the whole block because it is not just the mockserver that isn't required there anymore, correct?

Thought about it but then decided against it. I think it is not the responsibility of the mockserver configuration to clean up other, perhaps unrelated middlewares from `ui5.dependencies`.

Reply:
Agree.